tests/lib/venv.py

```diff
@@ -171,7 +170,7 @@ def _customize_site(self) -> None:
     def clear(self) -> None:
         self._create(clear=True)
 
-    def move(self, location: str) -> None:
+    def move(self, location: Union[Path, str]) -> None:
```

Comment:
error: Argument 1 to "move" has incompatible type "Path"; expected "str" 
see: https://github.com/python/typeshed/blob/91d6383d9d1ca38157ce46bb498a11347658db1d/stdlib/shutil.pyi#L105

Reply:
So it seems PathLike support here was only added in 3.9, and pre-commit missed this since we only run linters against the newest Python version.